docs/_docs/Instructors/02-Teachers-How-to-build-Learning-Units.md

```diff
@@ -1,160 +1,220 @@
-> 7. Have the reviewer go through your content. The reviewer should now build the Examples Notebook ("cheat-sheet")
-> 8. With all materials approved, build the presentation. Keep it short on text, think of how you will be telling the story, and what images can support it. 
+For most of the specializations and units, the materials from previous batches
+already have a high level of quality. Given that, our process assumes the 
+development will be done on top of the previous batch. 
 
-## Practicalities 
+## Instructors repository 
 
-#### Where should I push the work in progress? 
-- The Learning Units are pushed to the [Academy GitHub](https://github.com/LDSSA/), the repository depending on the edition of the Academy. 
-- Be particularly careful with the Exercise Notebook, as you don't want to push the answers. If in doubt, push things to your personal repo when working on them with your reviewer / team, and then push the right versions at the end.
+As mentioned in the [Teaching Process](./01-Teaching-process.md), the Learning Units 
+are pushed to the [Academy GitHub](https://github.com/LDSSA/), 
+to the repository with the name format `batchX-instructors`.
 
-#### How should I name my files? 
-The files should be named as follows: 
-> **[Learning / Examples / Exercise] Notebook - SLUx - [name of learning unit]**  
-> e.g. Learning Notebook - SLU6 - Intermediate Statistics
 
-#### Any specifics about the presentation? 
-Please use Google Slides, shared in anyone can view mode. Your link should be to the presentation itself, not the full screen mode, so that other people can clone it and change it. 
+Once the base improvements are done, the unit is shared with the QA team, which will 
+review and test the unit. This feedback helps us detect potential bugs and places for 
+improvement before the materials reach the students.
 
-#### But... can't I use [Keynote / Powerpoint / Pdf / RISE]? 
-It would be best if you didn't, so that it's easy for:
-- it to run on other people's computers during the presentations   
-- students to be able to see it without installing anything  
-- other people to copy it, change a slide or two, and have their presentation ready  
+### Adding Units
 
-## Mind your timeline 
+> ⚠️ Section in Progress 
 
-Building Learning Units **takes time!** You will probably take at least 8 hours to build every 1 hour of high quality learning material, so budget for it, and ensure you have a lot of time for revisions and corrections and that you send the materials to the QA Team by the agreed deadline.
+Suggested execution order:
+1. Carefully read the scope of your learning unit in the Curriculum repo, and think about what the students should be able to do after they have finished  
+2. Go through the learning notebooks and check for anything that needs improvements
+3. Google around and see how other people teach the concepts. 
+4. Submit the Learning Notebook and Example Notebook's improvements 
+5. [Optional] Share with QA  
```

Comment:
I'd remove this step for the sake of not over-burdening QA

Reply:
Can we discuss this as a separate issue? We need to align how QA will work now that the repo contains all the materials.

Reply:
Of course! :) Tagging @anaritarc for visibility